### ISSUE TITLE
Missing cursor hover values in 2d image plots

### ISSUE THREAD

**Issue by nvaytet** [scipp/plopp#47]
This is because we are now using `pcolormesh` instead of `imshow`, which used to give the value by default.

We should try to re-enable this, probably using something like https://stackoverflow.com/a/42581480/13086629

We could either have this enabled/disabled by a 'hover tool' in the toolbar, or have it enabled by default, as long as the number of pixels is small (because the searchsorted should be fast enough), and have as an opt-in for larger images.

**Comment by SimonHeybrock** [comment]
I feel we should up the priority of this. As it is, 2d plots can be quite useless in some cases. For example:

<img width="653" alt="image" src="https://user-images.githubusercontent.com/12912489/233612833-b9ecd68d-2de8-4495-85ac-391dadd44880.png">
<img width="654" alt="image" src="https://user-images.githubusercontent.com/12912489/233612885-5d753424-4e12-4300-b72e-beeb24cda0e6.png">

In linear scale I cannot tell what the "small" values are, and in log scale I cannot tell what any values are, in particular since the log range is relatively small.

**Comment by nvaytet** [comment]
@SimonHeybrock does this also need to work with 2d coords?

**Comment by SimonHeybrock** [comment]
> @SimonHeybrock does this also need to work with 2d coords?

Why not? Of course having it for 1d coords only initially would already be a big improvement.

**Comment by nvaytet** [comment]
Yes, I got it to work for both. I asked initially because the implementation I was thinking of was not going to work with 2d coords, but I found another way.
